tests/installation/ipxe_install.pm

```diff
@@ -70,13 +70,18 @@ sub set_bootscript {
     my $arch = get_required_var('ARCH');
     my $autoyast = get_var('AUTOYAST', '');
     my $mirror_http = get_required_var('MIRROR_HTTP');
+    my $openqa_url = get_var('OPENQA_URL', 'openqa.suse.de');
+    my $agama_iso = get_var('ISO');
 
     # trim all strings from variables to get rid of bogus whitespaces
     $arch =~ s/^\s+|\s+$//g;
     $autoyast =~ s/^\s+|\s+$//g;
     $mirror_http =~ s/^\s+|\s+$//g;
+    $openqa_url =~ s/^\s+|\s+$//g;
+    $agama_iso =~ s/^\s+|\s+$//g;
 
-    my $install = "install=$mirror_http";
+    my $agama_installer = get_var('AGAMA_INSTALLER', "http://$openqa_url/assets/iso/$agama_iso");
+    my $install = is_sle('>=16') ? "root=live:$agama_installer live.password=$testapi::password" : "install=$mirror_http";
```

Comment:
I haven't tried this yet... Just to double confirm. Have this way been reviewed by agama experts and they agree with it as a supported way for pxe installation?